events/Gondor.txt

```diff
@@ -86,9 +86,9 @@ country_event = {
 country_event = {
 	id = 50001
 	picture = "gondor/dol_amroth"
-	title = "Succession!"
+	title = "Secession!"
 	desc = "EVTDESC50001"
-	
+	major = yes
```

Comment:
You'll need to update the images to be major images (instead of the standard country events) -- the date location is different between standard and major country events, and will not look right with the wrong event image being used (applied to all events getting marked as major)